<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_paramconnexion</class>
 <widget class="QDialog" name="dlg_paramconnexion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>195</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>195</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>415</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Paramètrage de connexion à la base</string>
  </property>
  <widget class="QWidget" name="buttonsWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>390</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>390</width>
     <height>50</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="UpPushButton" name="AnnuluppushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
      <property name="icon">
       <iconset resource="assets/Images/images.qrc">
        <normaloff>:/Supprime.png</normaloff>:/Supprime.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>11</width>
        <height>11</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="UpPushButton" name="TestuppushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>Tester</string>
      </property>
      <property name="icon">
       <iconset resource="assets/Images/images.qrc">
        <normaloff>:/Test.ico</normaloff>:/Test.ico</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="UpPushButton" name="OKuppushButton">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="icon">
       <iconset resource="assets/Images/images.qrc">
        <normaloff>:/start.png</normaloff>:/start.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>390</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>390</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>390</width>
     <height>250</height>
    </size>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="MDPlabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Mot de passe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MDPlineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="PortcomboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
   </widget>
   <widget class="QLabel" name="Portlabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>75</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>211</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UpPushButton</class>
   <extends>QPushButton</extends>
   <header>uppushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>MDPlineEdit</tabstop>
  <tabstop>PortcomboBox</tabstop>
  <tabstop>AnnuluppushButton</tabstop>
  <tabstop>TestuppushButton</tabstop>
  <tabstop>OKuppushButton</tabstop>
 </tabstops>
 <resources>
  <include location="assets/Images/images.qrc"/>
 </resources>
 <connections/>
</ui>
